### PR TITLE
Fix readable_msisdn function logic

### PR DESCRIPTION
**Pull request by NicolaasDieBaas**

### CODE REVIEW

**Comment by NicolaasDieBaas** [comment]
@gsvr please review

File: test/test_utils.js [hunk 1]
```diff
@@ -74,6 +74,12 @@ describe("Testing utils functions", function() {
             assert.equal(utils.readable_msisdn("0821234567", "+27"), "0821234567");
             assert.equal(utils.readable_msisdn("+27821234567", "27"), "0821234567");
         });
+        // the next test case should not occur as the msisdn's passed in should
+        // either be normalized (start with '+') or as is (start with '0')
+        it("should return msisdn as is if contains no leading +", function() {
```
Comment (gsvr) [inline test/test_utils.js]: this sentence is wonky

File: test/test_utils.js [hunk 2]
```diff
@@ -74,6 +74,12 @@ describe("Testing utils functions", function() {
...
+            assert.equal(utils.readable_msisdn("27821234567", "+27"), "27821234567");
+            assert.equal(utils.readable_msisdn("27821234567", "27"), "27821234567");
```
Comment (gsvr) [inline test/test_utils.js]: should it not return `082...`?

Comment (NicolaasDieBaas) [inline test/test_utils.js]: it's open for discussion.  If the purpose of 'readable_msisdn' is to mainly be an inverse of the function that normalizes msisdn's, then we shouldn't ever get a msisdn passed in to 'readable_msisdn' that doesn't start with '+'.  Don't you agree?  But we might want to be able to deal with msisdn's like the one in question... What's your call; should I leave as is for now or improve?

Comment (gsvr) [inline test/test_utils.js]: I don't know what you just said - my head is too full of other stuff. What I do know is that `readable_msisdn` should present a readable msisdn - i.e. one without country code in the front. In South Africa we always want to present a 10-digit number starting with `0`. Can't think about other countries now - I'll leave that up to you.

**Comment by NicolaasDieBaas** [comment]
@gsvr please review.  Can we settle for now with the changes applied in last commit?

**Pull request merged**